content/c/concepts/pointers/pointers.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: "Pointers" 
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Pointers"
+  - "Memory"
+  - "Variables"
+  - "Arrays"
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A pointer is a [variable](https://www.codecademy.com/resources/docs/c/variables) that stores a memory address, which typically represents the location of another variable.
+
+Pointers are very useful in that they allow the efficient creation and manipulation of complex data structures.
+
+## Understanding Pointers
+
+Data is stored in a computer's [memory](https://www.codecademy.com/resources/docs/general/what-is-ram). A variable represents a specific location within that memory, and consists of three parts: an identifier (name), a value, and an *address*. The address is a value that describes *where* in memory a variable is located.
```

Comment:
Data is stored in a computer's [memory](https://www.codecademy.com/resources/docs/general/what-is-ram). A variable represents a specific location within that memory and consists of three parts: 

- An identifier (name)
- A value
- An address

The address is a value that describes *where* in the memory the variable is located.